docs/set-minimal-iso.md

```diff
@@ -46,9 +46,9 @@ function log() {
 trap log EXIT
 
 echo Telling service to generate a minimal ISO with our public SSH key file
-curl -fail -s $OCM_API_ENDPOINT/assisted-install/v1/clusters/$CLUSTER_ID/downloads/image -H "Authorization: Bearer $TOKEN" --request POST --header "Content-Type: application/json" --data @<(echo '{"image_type": "minimal-iso", "ssh_public_key": ""}' | jq --rawfile pubkey <(echo -n $SSH_KEY) '.ssh_public_key = $pubkey')
+curl -fail -s $OCM_API_ENDPOINT/assisted-install/v2/infra-envs/$INFRA_ENV_ID -H "Authorization: Bearer $TOKEN" --request POST --header "Content-Type: application/json" --data @<(echo '{"image_type": "minimal-iso", "ssh_authorized_key": ""}' | jq --rawfile pubkey <(echo -n $SSH_KEY) '.ssh_authorized_key = $pubkey')
```

Comment:
shouldn't line 29 change as well? (i can't comment it)
```
if [ -z ${INFRA_ENV_ID+x} ]; then
	echo 'Please set INFRA_ENV_ID to your infra-env ID, which can be found in the Assisted Installer URL'
	exit 1
fi
```

Reply:
Done